routes/orders.js

```diff
@@ -73,24 +73,24 @@ ordersRouter.post("/", requireUser, async (req, res, next) => {
     }
 });
 
-ordersRouter.patch("/:orderId", async (req, res, next) => {
+ordersRouter.patch("/:orderId", requireUser, async (req, res, next) => {
 
```

Comment:
needs requireAdmin

Reply:
I thought we agreed we were just going to leave it like this for now so the user could update the order at checkout to 'completed'.

Reply:
Has this issue been resolved?